docs/sdk/examples/exportpdf.html

```diff
@@ -0,0 +1,179 @@
+
+				<p>Angelina Calvino, food journalist</p>
+			</blockquote>
+
+			<p>Please arrive at the <a href="https://ckeditor.com">Bilancino Hotel</a> reception desk at least <strong>half an hour earlier</strong> to make sure that the registration process goes as smoothly as possible.</p>
+
+			<p>We look forward to welcoming you to the event.</p>
+
+			<p><img alt="Victoria Valc signature" src="https://ckeditor.com/docs/ckeditor4/latest/examples/assets/image/signature.png" style="height:101px;width:180px" /></p>
+
+			<p><span style="font-size:16px"><strong>Victoria Valc</strong></span></p>
+
+			<p><strong>Event Manager<br />
+			Bilancino Hotel</strong></p>
+
+			<p>&nbsp;</p>
+
+			<div style="page-break-after: always"><span style="display:none">&nbsp;</span></div>
+
+			<p>&nbsp;</p>
+
+			<h2 style="text-align:center"><span style="font-family:Georgia,serif"><span style="color:#2980b9">The Flavorful Tuscany Meetup Schedule</span></span></h2>
+
+			<table align="center" border="1" cellspacing="0" style="border-collapse:collapse; width:597px">
+				<thead>
+					<tr>
+						<th colspan="2" style="background-color: rgb(153, 153, 153);"><span style="color:#ffffff">Saturday, July 14</span></th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<td style="background-color:#e6e6e6; text-align:center">9:30 AM - 11:30 AM</td>
+						<td>
+							<p style="text-align:center"><strong>Americano vs. Brewed - “know your coffee”</strong> with:&nbsp;</p>
+
+							<ul>
+								<li>Giulia Bianchi</li>
+								<li>Stefano Garau</li>
+								<li>Giuseppe Russo</li>
+							</ul>
+						</td>
+					</tr>
+					<tr>
+						<td style="background-color:#e6e6e6; text-align:center">1:00 PM - 3:00 PM</td>
+						<td>
+							<p style="text-align:center"><strong>Pappardelle al pomodoro</strong> - live cooking&nbsp;<sup>1</sup></p>
+
+							<p style="text-align:center">Incorporate the freshest ingredients&nbsp;<br />
+							with Rita Fresco</p>
+						</td>
+					</tr>
+					<tr>
+						<td style="background-color:#e6e6e6; text-align:center">5:00 PM - 8:00 PM</td>
+						<td>
+							<p style="text-align:center"><strong>Tuscan vineyards at a glance</strong> - wine-tasting&nbsp;<br />
+							with Frederico Riscoli</p>
+						</td>
+					</tr>
+				</tbody>
+			</table>
+		</textarea>
+
+		<h2 id="related-features">Related Features</h2>
+		<ul>
+			<li><a href="./pastefromword.html">Inserting Content &ndash; Pasting Content from Microsoft Word</a></li>
+			<li><a href="./size.html">Editor Resizing - Setting Editor Size</a></li>
+			<li><a href="./acf.html">Content Filtering – Advanced Content Filter - Automatic Mode</a></li>
+		</ul>
+
+		<script>
+			// CKEDITOR.plugins.addExternal( 'pdfexport', '../../../ckeditor4/{%CKEDITOR_VERSION%}/{%CKEDITOR_EXAMPLES_SLUG%}/assets/plugins/pdfexport/', 'plugin.js' );
+
+			if ( CKEDITOR.env.ie && !CKEDITOR.env.version < 11 ) {
```

Comment:
There is something wrong with this condition, should it be:

```javascript
CKEDITOR.env.ie && CKEDITOR.env.version < 11
```

without (`!`)?